bench-tps/src/bench.rs

```diff
@@ -630,9 +641,13 @@ pub fn generate_keypairs(id: &Keypair, tx_count: usize) -> Vec<Keypair> {
 
     let mut total_keys = 0;
     let mut target = tx_count * 2;
-    while target > 0 {
+    while target > 1 {
         total_keys += target;
-        target /= MAX_SPENDS_PER_TX;
+        // Use the upper bound for this division otherwise it may not generate enough keys
+        target = match target.checked_rem(MAX_SPENDS_PER_TX) {
+            Some(0) => target / MAX_SPENDS_PER_TX,
```

Comment:
can you just do `(target + MAX_SPENDS_PER_TX + 1) / MAX_SPENDS_PER_TX`

Reply:
yup this should work.